src/cli.ts

```diff
@@ -4,18 +4,19 @@ import fs = require('fs');
 import size_tree = require('./size_tree');
 import webpack_stats = require('./webpack_stats');
 
-function printStats(json: string, outputAsJson: boolean) {
+function printStats(json: string, opts: { outputAsJson: boolean, sharesStat: boolean }) {
     const bundleStats = JSON.parse(json) as webpack_stats.WebpackStats;
 	const depTrees = size_tree.dependencySizeTree(bundleStats);
-    if (outputAsJson) {
+    if (opts.outputAsJson) {
         console.log(JSON.stringify(depTrees, undefined, 2));
     } else {
-        depTrees.forEach(tree => size_tree.printDependencySizeTree(tree));
+        depTrees.forEach(tree => size_tree.printDependencySizeTree(tree, opts.sharesStat));
     }
 }
 
 commander.version('1.1.0')
          .option('-j --json', 'Output as JSON')
+         .option('--no-shares-stat', 'Suppress shares stat from output')
```

Comment:
I think the wording of this option would sound better as `--no-share-stats` (ie. it suppresses multiple statistics, of one particular type).

Also, I think it might not be clear from the description what this option does. Perhaps "Do not output dependency sizes as a percentage"?